requirements.txt

```diff
@@ -10,3 +10,4 @@ cryptography
 requests
 av
 numpy
+pycaw
```

Comment:
is there a way to make this a requirement only for windows?

Reply:
not an expert on this file, but either a separate one for windows (not great), or just extend the pip command in the readme to e.g.:

`pip install -r requirements.txt pycaw`

Could you test whether that works?

Reply:
yes, that works. so may be just modifying readme instead of adding to requirements.txt

Reply:
Feel free to PR my branch - if not, I'll do it when I'm next at my PC.

Reply:
Sorted it.